DotNet/overrides.js

```diff
@@ -67,4 +67,5 @@
 	'check region by element within shadow dom with vg': {skipEmit: true},
 	'pageCoverage data is correct': {skipEmit: true},
     'pageCoverage data is correct with vg': {skipEmit: true},
+		'AUTproxy should proxy resources test manually only': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8974587)
```suggestion
		"AUTproxy should proxy resources test manually only": {skipEmit: true},
```